docs/src/modules/components/Ad.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Tooltip from '@material-ui/core/Tooltip';
+import withWidth from '@material-ui/core/withWidth';
+import CodeFund from 'docs/src/modules/components/CodeFund';
+import CarbonAds from 'docs/src/modules/components/CarbonAds';
```

Comment:
Just Carbon?

Reply:
Ha, that's what I had originally 'till I looked at their site. 😄 

I'll change it back.